app/dao/inbound_numbers_dao.py

```diff
@@ -4,7 +4,7 @@
 
 
 def dao_get_inbound_numbers():
-    return InboundNumber.query.all()
+    return InboundNumber.query.order_by(InboundNumber.updated_at, InboundNumber.number).all()
```

Comment:
I would order the query by one or the other. I would as @pete what order he prefers.